### DISCUSSION TITLE
[bug] gem5 to tlm build error (/util/tlm/SConstruct)

### DISCUSSION THREAD

**Discussion by gxflying** [gem5/gem5#491] [gem5-dev]
hi sir I have clone the gem5 release version  gem5-23.0.1.0，  when I try to execute the cmd : "scons " in the path $(GEM5_ROOT)/util/tlm
       it shows the wrror below
**AttributeError: 'Values' object has no attribute 'duplicate_sources':**

and I fix the this error by move the L85 to the very front of the tlm/SConstruct
and it seems that the option should be added before it is used

https://github.com/gem5/gem5/blob/1bb9bb33086aec5f39fdc0148552d01e29326cb0/util/tlm/SConstruct#L85

please help to clarify this,  thanks

**Comment by BobbyRBruce** [comment]
Reported in Issue #591.

Definitely a bug we need to fix. Thanks for reporting.
